.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,19 @@
+# Contributing to MNE-Python
+
+First off, thanks for taking the time to contribute!
+
+The following is a quick summary to a set of guidelines for contributing to [MNE-Python](https://github.com/mne-python/mne-python) on GitHub. These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+#### Table Of Contents
```

Comment:
I think we can skip the TOC in such a short document.

Reply:
agreed